lib/react/renderer.rb

```diff
@@ -22,9 +23,9 @@ def self.setup!(react_js, components_js, args={})
       @@pool = ConnectionPool.new(default_pool_options.merge(args)) { self.new }
     end
 
-    def self.render(component, args={})
+    def self.render(component, args={}, replay_console=false)
```

Comment:
Could `replay_console` be a member of the `args` hash? That would be nice because it would keep the Ruby idiom of options-hash-last.

I think you could fetch it out with: 

```ruby
replay_console =  !!args.delete(:replay_console)
```